Problems/1966/ans_1966_JHP.py

```diff
@@ -0,0 +1,35 @@
+# https://www.acmicpc.net/problem/1966
+# 문제 : 프린터큐
+# 입력 : 첫줄 = TC 수, 
+#      : TC 첫줄 = [문서 개수 N, 궁금한 문서가 현재 Queue에 몇 번째인지 M]
+#      : TC 둘째줄 = N개 문서의 중요도 
+# 출력 : 각 TC별 문서가 몇번째 출력 되는지 확인
+
+import sys
+
+def solve():
+    input = sys.stdin.readline
+    TC = int(input())
+    
+    for _ in range(TC):
+        N,M = map(int,input().split())
+        input_doc = list(map(int,input().split()))
+        result = [0]*N
+        result[M] = 'True'
```

Comment:
```suggestion
        result = [False]*N
        result[M] = True
```
리스트가 type에 자유롭긴 하지만, 섞어쓰면 좋지 않을 것.
의도를 봐서는 Boolean (True)를 쓰고싶었던 것으로 보이는데, 따옴표 쓰면 string이 됨.